client/components/login/login.js

```diff
@@ -345,7 +345,7 @@ Login.propTypes = {
       links: PropTypes.arrayOf(PropTypes.object),
     }),
     input_fields: PropTypes.shape({
-      username: PropTypes.object,
+      email: PropTypes.object,
```

Comment:
@nikitaermishin can you please make it more detailed using PropTypes.shape(). Like
```
email : PropTypes.shape({
type: PropType.string.isRequired
...
})
```
In this way, we can make sure a required field is defined and supplied to the component. For input fields only `type` will be required and all others will be optional.